### DISCUSSION TITLE
CSS doesn’t apply to my html

### DISCUSSION THREAD

**Discussion by a-mds90** [community/community#23232] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone!<br>
I’m pretty new at this and I can’t see what I’m doing wrong.<br>
The html seems to load, but the css won’t apply at all.<br>
Can someone help?</p>
<p>The github page : <a href="https://a-mds90.github.io/coursera-web-dev-course/mod-2-solution/index.html" class="inline-onebox" rel="noopener nofollow ugc">Module 2 Assignment</a><br>
My files : <a href="https://github.com/a-mds90/coursera-web-dev-course/tree/master/mod-2-solution" class="inline-onebox" rel="noopener nofollow ugc">coursera-web-dev-course/mod-2-solution at master · a-mds90/coursera-web-dev-course · GitHub</a></p>

**Comment by kingthorin** [comment, marked as answer]
<p>I assume you mean here? <a href="https://a-mds90.github.io/coursera-web-dev-course/mod-2-solution/" class="inline-onebox" rel="noopener nofollow ugc">Module 2 Assignment</a></p>
<p>I think the issue is the style tags surrounding the link tag.</p>

**Comment by a-mds90** [comment]
<p>Thanks! That was indeed the issue!</p>

**Comment by kingthorin** [comment]
<p>Great, glad you got it sorted out.</p>
